examples/features/broker-connection/disaster-recovery/readme.md

```diff
@@ -0,0 +1,11 @@
+# AMQP Broker Connection with Senders and SSL
```

Comment:
yeah.. it is not done yet. thanks

Reply:
Its hard to tell what is isnt, so its just easier to comment when spotting things rather than needing to go back and look again later, and better than getting forgotten about if noone does.